usaspending_api/accounts/serializers.py

```diff
@@ -126,7 +126,6 @@ class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
 
 class AgenciesFinancialBalancesSerializer(serializers.Serializer):
 
-    fiscal_year = serializers.IntegerField()
```

Comment:
I'm all for leaving it as is for readability. So if it's removed from the serializer, but not the queryset, it won't be part of the response at all?